src/validation/repair.jl

```diff
@@ -3,19 +3,19 @@
 # Released under the modified BSD license. See COPYING.md for more details.
 
 """
-    repair!(instance)
+    repair!(sc)
 
-Verifies that the given unit commitment instance is valid and automatically
+Verifies that the given unit commitment scenario is valid and automatically
 fixes some validation errors if possible, issuing a warning for each error
 found. If a validation error cannot be automatically fixed, issues an
 exception.
 
 Returns the number of validation errors found.
 """
-function repair!(instance::UnitCommitmentInstance)::Int
+function repair!(sc::UnitCommitmentScenario)::Int
```

Comment:
As discussed, it's fine to change this one to `repair!(::UnitCommitmentScenario)` but I think we should still have another one with signature `repair!(::UnitCommitmentInstance)`, which just calls this one.